.bazelrc

```diff
@@ -41,6 +42,9 @@ build:release-ios --apple_bitcode=embedded
 build:release-ios --config=ios
 build:release-ios --config=release-common
 build:release-ios --copt=-fembed-bitcode
+# This is a workaround because we can't set --compilation_mode=opt because of size issues
+# this forces -O to be passed at least to Swift compiles
```

Comment:
> we can't set --compilation_mode=opt because of size issues

is this for c++ code that it causes a problem? Is it because `opt` does flags other than `-Os`?

Reply:
not sure, but the resulting binary is too large for zip